### PR TITLE
Create a Doc Page for the SingleStore Data Integration

### PR DESCRIPTION
**Pull request by Sriparno08**
## Description

I have created a doc page for the SingleStore data integration.

**Fixes** #4999

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 📄 This change requires a documentation update

### What is the solution?

I have created a doc page for the SingleStore data integration.

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

File: docs/data-integrations/singlestore.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+The required arguments to establish a connection are,
+* `host`: host name or IP address
+* `port`: port used to make TCP/IP connection
+* `database`: database name
+* `user`: database user
+* `password`: database password
+
+## Usage
+In order to make use of this handler and connect to Databricks in MindsDB, the following syntax can be used,
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/data-integrations/singlestore.mdx:25]: The code block should be as below:

> \```sql
> SQL GOES HERE
> \```

File: docs/data-integrations/singlestore.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+The required arguments to establish a connection are,
+* `host`: host name or IP address
+* `port`: port used to make TCP/IP connection
+* `database`: database name
+* `user`: database user
+* `password`: database password
+
+## Usage
+In order to make use of this handler and connect to Databricks in MindsDB, the following syntax can be used,
```
Comment (martyna-mindsdb) [inline docs/data-integrations/singlestore.mdx:24]: It is not about Databricks. Please correct it.

**Comment by martyna-mindsdb** [comment]
@Sriparno08 
Please see the review comments above and implement the changes in this PR.

**Comment by Sriparno08** [comment]
Okay, @martyna-mindsdb 👍

**Pull request merged**